resteasy/src/test/scala/com/cerner/beadledom/resteasy/ResteasyServiceSpec.scala

```diff
@@ -420,64 +419,6 @@ class ResteasyServiceSpec(rootUrl: String, tomcatPort: Int)
       }
     }
 
-    describe("Swagger") {
```

Comment:
Are there similar tests for swagger 2?

Reply:
I haven't found any. I added some back with 20b4717.